Makefile

```diff
@@ -1,6 +1,6 @@
-VERSION ?= 1.0.0-SNAPSHOT
+VERSION = $(or $(shell git describe --exact-match HEAD 2> /dev/null), latest)
```

Comment:
We shouldn't be using "latest". I'd recommend always using an image tag that refers to an immutable bunch of bits. The git SHA1 or most recent tag works well for this:

`git describe --tags --always --dirty`

Reply:
gotcha, I updated the version generation to use `git describe --tags --always --dirty`